docs/usage/pod-security.md

```diff
@@ -67,4 +67,4 @@ For proper functioning of Gardener, `kube-system` namespace will also be automat
 
 If this field is set to `true`, then all authenticated users can use the "gardener.privileged" `PodSecurityPolicy`, allowing full unrestricted access to Pod features. However, the `PodSecurityPolicy` admission plugin is removed in Kubernetes `v1.25` and `PodSecurity` has taken its place as its successor. Therefore, this field doesn't have any relevance in versions `>= v1.25` anymore. If you need to set a default pod admission level for your cluster, follow [this documentation](#admission-configuration-for-the-podsecurity-admission-plugin).
 
-> **Note:** You should remove this field from the `Shoot` spec for `v1.24` clusters after migrating to the new `PodSecurity` admission controller, before upgrading your cluster to `v1.25`.
+> **Note:** You should remove this field from the `Shoot` spec for `v1.24` clusters only after migrating to the new `PodSecurity` admission controller, ie; after disabling the old `PodSecurityPolicy` admission plugin (see [Migrating from `PodSecurityPolicy`s to PodSecurity Admission Controller](#migrating-from-podsecuritypolicys-to-podsecurity-admission-controller)). Otherwise the field is defaulted by the Gardener API Server. This is intentional because if we allow the field to be removed when `PodSecurityPolicy` admission plugin is active, the existing pods running in the clusters which needs privileges can fail.
```

Comment:
```suggestion
> **Note:** You should remove this field from the `Shoot` spec for `v1.24` clusters only after migrating to the new `PodSecurity` admission controller, i.e. after disabling the old `PodSecurityPolicy` admission plugin (see [Migrating from `PodSecurityPolicy`s to PodSecurity Admission Controller](#migrating-from-podsecuritypolicys-to-podsecurity-admission-controller)). Otherwise the field is defaulted by the Gardener API Server. This is intentional because if we allow the field to be removed when `PodSecurityPolicy` admission plugin is active, the existing pods running in the clusters which need privileges can fail.
```